migtests/scripts/functions.sh

```diff
@@ -909,23 +909,22 @@ normalize_json() {
         end
     )' "$input_file" > "$temp_file"
 
-	# Sample code: how to ignore nested fields with a specific parent in a json
-	# TODO: For fields like SqlStatement or others we can more explicit regarding changing them
-    # jq --sort-keys 'walk(
-    #     if type == "object" and (.UnsupportedFeatures?|type) == "array" then
-	# 		.UnsupportedFeatures |= map(
-	# 			.FeatureName? = "IGNORED"
-	# 		)
-    #     else
-	# 		.
-    #     end
-    # )' "$temp_file" > "$temp_file2"
+	# Second pass: for AssessmentIssue objects, if Category is "migration_caveats" and the Type is one
+    # of the two specified values, then set ObjectName to "IGNORED".
+    jq 'walk(
+        if type == "object" and
+			.Category == "migration_caveats" and
+			(.Type == "UNSUPPORTED_DATATYPE_LIVE_MIGRATION" or .Type == "UNSUPPORTED_DATATYPE_LIVE_MIGRATION_WITH_FF_FB")
+        then .ObjectName = "IGNORED"
```

Comment:
Why is it required to ignore the object name for these Type?

Reply:
I think just a temporary ignore until we fix it properly :)

Reply:
yes, plan is to have individual issue for each datatype. 
Updating the test for this temporary fix will be a big change, a lot of test has the corresponding issues including osm, omnibus

Reply:
oh okay, got it!